packages/boxed-expression-component/src/contextMenu/MenuWithHelp/MenuItemWithHelp.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import "./MenuWithHelp.css";
+import { MenuItem, MenuItemAction } from "@patternfly/react-core/dist/js/components/Menu";
+import { HelpIcon } from "@patternfly/react-icons/dist/js/icons/help-icon";
+
+export interface MenuItemWithHelpProps {
+  menuItemKey: string;
+  menuItemCustomText?: string;
+  menuItemDescription: string;
+  menuItemIcon?: any;
+  menuItemIconStyle?: any;
+  setVisibleDescription: (description: string) => void;
+  getVisibleDescription: () => string;
```

Comment:
This doesn't need to be a function, can be a plain value.


```suggestion
  visibleDescription: string;
```

Reply:
done as 1e2bfcd372